src/components/createActivityForm/CreateActivityForm.spec.js

```diff
@@ -0,0 +1,124 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import CreateActivityForm from './CreateActivityForm'
+
+describe('CreateActivityForm', () => {
+  it('has ten required input fields', () => {
+    const mockOnCreateNewActivity = jest.fn()
+    render(<CreateActivityForm onCreateNewActivity={mockOnCreateNewActivity} />)
```

Comment:
You only need a mock function, when you actually mock something. When I see a mock function I would expect to see some interaction in the test.

When all you need is a function and it is never really called, you could make it clear like this
```jsx
const noOp = () => {}
render(<CreateActivityForm onCreateNewActivity={noOp} />)
```

Same goes for the mock functions below.